api/types.ts

```diff
@@ -58,6 +75,20 @@ export interface LCQuestion {
   url: string;
 }
 
+/**
+ * Questions is a map of questions by question name.
+ */
+export interface Questions {
+  [question_name: string]: Question;
+}
+
+/**
+ * Scores is a map of scores by Discord user ID.
+ */
+export interface Scores {
+  [discord_user_id: string]: number;
+}
+
 /**
  * Season is a season of the leaderboard.
```

Comment:
Resolve #44.

Reply:
Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/41/commits/1b1890ffa2cdcad24ba9bfa2473e465ceae19d9c>.